src/main/java/com/training/qlsv/service/CourseService.java

```diff
@@ -0,0 +1,61 @@
+package com.training.qlsv.service;
+
+import com.training.qlsv.dao.CourseDao;
+import com.training.qlsv.model.Course;
+import com.training.qlsv.model.Student;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class CourseService {
```

Comment:
chuyển toàn bộ các phần nhập từ bàn phím ra ngoài, không viết trong service, service chỉ xử lý logic